packages/gitgraph-node/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { GitgraphCore } from "gitgraph-core/lib";
+
+// Domain (rendering logic)
+import computeGraphMap from "./compute-graph-map";
+
+// Infrastructure (logger implementations)
+import consoleGraphLogger from "./console-graph-logger";
+
+const gitgraph = new GitgraphCore();
```

Comment:
With this pattern, we can't personalize the template (eg: colors in this particular usecase) I will prefer to give a gitgraph instance to render to avoid this lack of config